source/main.ts

```diff
@@ -40,7 +40,7 @@ global.log = {
 
 // We need the app and process modules.
 const { app } = require('electron')
-const process = require('process')
+const processInfo = require('process')
```

Comment:
Why did you rename this as processInfo?

Reply:
Typescript complained that `process` was already defined elsewhere... (didn't investigate where)

Reply:
Huh, then we should investigate that, because `processInfo` seems like a misnomer as it doesn't only contain information …